docs/DESIGN_COMPARISON.md

```
@@ -0,0 +1,523 @@
+  retry:
+    max_attempts: 3
+    initial_delay: 100ms
+```
+
+---
+
+## Architecture Comparison
+
+### NG (Full) - Enterprise Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)